dds/DCPS/RTPS/rtps.mpc

```diff
@@ -1,4 +1,4 @@
-project(OpenDDS_Rtps): dcpslib, install, rtps_optional_safety {
+project(OpenDDS_Rtps): dcpslib, install, rtps_optional_safety, msvc_bigobj {
```

Comment:
Just noting that we may want to revisit this if the default is to generate minimal TypeObjects only.